assets/src/edit-story/components/panels/test/textStyle.js

```diff
@@ -237,21 +237,36 @@ describe('Panels/TextStyle', () => {
     });
 
     it('should update horizontal padding with lock', () => {
-      const { getByRole, pushUpdateForObject } = renderTextStyle([textElement]);
+      const { getByRole, pushUpdateForObject, pushUpdate } = renderTextStyle([
+        textElement,
+      ]);
       const input = getByRole('textbox', {
         name: 'Edit: Horizontal & Vertical padding',
       });
-      fireEvent.change(input, { target: { value: '11' } });
+      fireEvent.change(input, { target: { value: '20' } });
       expect(pushUpdateForObject).toHaveBeenCalledWith(
         'padding',
-        { horizontal: 11, vertical: 11 },
+        { horizontal: 20, vertical: 20 },
         DEFAULT_PADDING,
         false
       );
+      const updaterFunction = pushUpdate.mock.calls[0][0];
+      const updatedContent = updaterFunction({
+        x: 40,
+        y: 40,
+        width: 100,
+        height: 100,
+      });
+      expect(updatedContent).toStrictEqual({
+        x: 20,
+        y: 20,
+        width: 140,
+        height: 140,
```

Comment:
This is to test the new frame metrics.

New padding is `20`. Original is `0`. So `40 - 20 = 20` which results in `x` and `y` being both `20`.

Both `height` and `width` are `100`. `100 + (20 - 0) * 2 = 140` which results in both `width` and `height` being `140`.